style.css

```diff
@@ -1,4 +1,624 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400&family=Poppins:wght@400;500;700;800&display=swap');
 
-h1 {
-  color: aqua;
+:root {
+  --color-primary: #272a31;
+  --color-white: #fff;
+  --color-black: #000;
+  --color-secondary: #ec5242;
+}
+
+*,
+*::after,
+*::before,
+ul {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  color: var(--color-white);
+  font-size: 1.7rem;
+  line-height: 1.5;
+  font-weight: 400;
+}
+
+.row-style {
+  width: 88% !important;
+  margin: 0 auto !important;
+}
+
+.header {
+  position: sticky;
+  top: 0;
+  left: 0;
+  background-color: #fff;
+  box-shadow: 0 0.5rem 1rem 0.2rem rgb(0 0 0 / 5%);
+  z-index: 999;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please, the `hamburger` button must be inside the main page, that is, like the `hamburger` button shown in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), your `hamburger` button is highlighted with a `white background` at the `top` of the main page, please remove the `white` background and show the `hamburger` button on the main page without any `background`, I believe it will be very easy for you 😃 

![image](https://user-images.githubusercontent.com/63915024/198377670-0b4ff32f-4a10-4654-8600-86aff1258d96.png)

Reply:
Hello @mukhammadrizooff , thanks a lot for your feedbacks. However, I have seen other projects get accepted even though they have not strictly followed the design of the given hamburger menu. I am currently facing difficulties with making the changes you have said because it is my first time using bootstrap.